autopts/pybtp/defs.py

```diff
@@ -49,6 +49,7 @@ def BIT(bit):
 BTP_SERVICE_ID_HAP = 24
 BTP_SERVICE_ID_CSIP = 25
 BTP_SERVICE_ID_CAP = 26
+BTP_SERVICE_ID_TMAP = 27
```

Comment:
Let's use ID 28, as there are approved PRs with TBS support that uses ID 27
```suggestion
BTP_SERVICE_ID_TMAP = 28
```